src/ol/format/wkt.js

```diff
@@ -194,13 +199,15 @@ ol.format.WKT.GeometryEncoder_ = {
 /**
  * Parse a WKT string.
  * @param {string} wkt WKT string.
- * @return {ol.geom.Geometry|ol.geom.GeometryCollection|undefined}
+ * @return {ol.geom.SimpleGeometry|ol.geom.GeometryCollection|undefined}
  *     The geometry created.
  * @private
  */
 ol.format.WKT.prototype.parse_ = function(wkt) {
+  var dimensions = wkt.indexOf('Z') === -1 ? 2 : 3;
```

Comment:
Yeah, but there is no code that handles it. So I thought it was more clear if there wasn't semi-support.